submissions/rubyAdeniregun/Database/Queries.sql

```diff
@@ -0,0 +1,30 @@
+
+/* Question 1: Query to find the total number of passengers that survived 
+The total number of passengers who survived = 342. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 1; 
+
+
+/* Question2: Query to find the total number of passengers who did not survive
+The total number of passengers who did not survive = 549. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 0; 
+
+
+/* Question 3: Query to get the name and sex of passengers under 
+the age of 27 that embarked at Queenstown and Cherbourg? */
+SELECT registered_passengers.full_name, registered_passengers.sex, registered_passengers.age < '27', passengers_records.point_of_embarkation = 'C' AND 'Q'
```

Comment:
What you did works fine but, that's not what is expected. We only want to see 2 columns i.e `full_name` and `sex`. This `passengers under the age of 27 that embarked at Queenstown and Cherbourg?` should be on the `WHERE ` clause